src/Compatibility/GenAPI/Microsoft.DotNet.GenAPI/CSharpFileBuilder.cs

```diff
@@ -56,13 +61,21 @@ public CSharpFileBuilder(ILog logger,
 
         /// <inheritdoc />
         public void WriteAssembly(IAssemblySymbol assemblySymbol)
```

Comment:
I'm thinking that the CSharpFileBuilder class should just have a WriteAssembly method. The functionality from the methods that are inherited, since it's shared between GenAPIApp, the differ and the tests, should probably be moved to a different class. To make it generic (C# or VB in the future) I might have to use an interface as well.

I'm going to think about this more.